docs/source/install.rst

```diff
@@ -414,6 +414,14 @@ All cached entries can be viewed using ``cmake -LAH`` from a build directory.
     Path to an existing directory used to cache :file:`proj.db` to speed-up
     subsequent builds without modifications to source SQL files.
 
+.. option:: EMBED_PROJ_DATA_PATH
+
+    .. versionadded:: 9.5
+
+    Embed ``PROJ_DATA_PATH`` data files location as an alternative paths for
+    external libraries. Disable to avoid setting this non-relocatable hardcoded
+    paths. Default ON.
```

Comment:
```suggestion
    Embed ``PROJ_DATA`` hard-coded alternative path for data files location. Disable to avoid setting this non-relocatable hard-coded paths. Default ON.
```